src/OnlineSales/Program.cs

```diff
@@ -434,4 +450,58 @@ private static void ConfigureCORS(WebApplicationBuilder builder)
             });
         });
     }
+
+    private static void ConfigureIdentity(WebApplicationBuilder builder)
+    {
+        builder.Services.AddIdentity<User, IdentityRole>()
+            .AddEntityFrameworkStores<PgDbContext>();
+        builder.Services.ConfigureApplicationCookie(options =>
+        {
+            // Cookie settings
+            options.Cookie.HttpOnly = true;
+            options.ExpireTimeSpan = TimeSpan.FromHours(12);
+
+            options.LoginPath = "/api/Identity/ExternalLogin";
+            options.AccessDeniedPath = "/AccessDenied";
+            options.SlidingExpiration = true;
+        });
+
+        builder.Services.AddAuthentication(options =>
+        {
+            options.DefaultAuthenticateScheme = JwtBearerDefaults.AuthenticationScheme;
+            options.DefaultChallengeScheme = JwtBearerDefaults.AuthenticationScheme;
+        })
+            .AddMicrosoftIdentityWebApi(
+             jwtOptions =>
+             {
+                 jwtOptions.Events = new AzureAdJwtBearerEventsHandler();
+             }, identityOptions =>
+             {
+                 identityOptions.Instance = builder.Configuration.GetValue<string>("AzureAD:Instance") ?? string.Empty;
+                 identityOptions.TenantId = builder.Configuration.GetValue<string>("AzureAD:TenantId") ?? string.Empty;
+                 identityOptions.Domain = builder.Configuration.GetValue<string>("AzureAD:Domain") ?? string.Empty;
+                 identityOptions.ClientId = builder.Configuration.GetValue<string>("AzureAD:ClientId") ?? string.Empty;
+                 identityOptions.ClientSecret = builder.Configuration.GetValue<string>("AzureAD:ClientSecret") ?? string.Empty;
+             });
+        builder.Services.AddAuthentication().AddMicrosoftIdentityWebApp(
+            identityOptions =>
+            {
+                identityOptions.Instance = builder.Configuration.GetValue<string>("AzureAD:Instance") ?? string.Empty;
+                identityOptions.TenantId = builder.Configuration.GetValue<string>("AzureAD:TenantId") ?? string.Empty;
+                identityOptions.Domain = builder.Configuration.GetValue<string>("AzureAD:Domain") ?? string.Empty;
+                identityOptions.ClientId = builder.Configuration.GetValue<string>("AzureAD:ClientId") ?? string.Empty;
+                identityOptions.ClientSecret = builder.Configuration.GetValue<string>("AzureAD:ClientSecret") ?? string.Empty;
+                identityOptions.CallbackPath = "/api/Identity/Callback";
```

Comment:
Following our naming convention and the way how it is displayed in Swagger UI it should be /api/identity/callback